public/content/translations/de/developers/tutorials/how-to-write-and-deploy-an-nft/index.md

```diff
@@ -49,7 +49,7 @@ Zum Versenden und Empfangen von Transaktionen benötigen Sie ein Ethereum-Konto.
 
 Sie können [hier](https://metamask.io/download.html) MetaMask kostenlos herunterladen und ein Konto erstellen. Wie Sie ein neues Konto erstellen oder wenn Sie bereits ein Konto haben, stellen Sie bitte sicher, dass Sie zum Ropsten-Testnet oben rechts wechseln (um sicherzustellen, dass Sie nicht mit echtem Geld handeln).
 
-![Ropsten als Netzwerk festlegen](./metamask-ropsten.png)
+![Ropsten als Netzwerk festlegen](./metamask-goerli.png)
```

Comment:
This is a bug in our main repo, that I found in this repo because of the builds breaking. These are the correct new images, verified with the English tutorial. This happened when changing content from Ropsten which was deprecated to Goerli (which is now also deprecated 😅 but was right at the time)

Reply:
Haha ok, got it. I guess we now need to implement these fixes in the other repo as well right?